timecamp_sync_users.py

```diff
@@ -298,6 +298,27 @@ def _update_existing_user(self, existing_user: Dict[str, Any],
         else:
             logger.debug(f"Skipping role update for user {email} due to disable_role_updates config")
         
+        # Check if user needs to be re-enabled
+        # Force check fresh status
+        fresh_settings = self.api.get_user_settings([user_id], 'disabled_user')
```

Comment:
Potentially can slow the script if hundreds of users. Need to check if we don't have already this information.